### PR TITLE
OCPQE-14026: Adding jobs for 134_IPI on AWS & KMS & Etcd Encryption.install.yaml 

### PR DESCRIPTION
**Pull request by yunjiang29**
Adding `aws-ipi-byo-kms-etcd-encryption` to:
```
release-4.10__arm64-nightly
release-4.10__amd64-nightly
release-4.10__amd64-nightly
release-4.10__amd64-nightly
release-4.10__amd64-stable
release-4.10__amd64-stable
release-4.10__arm64-nightly
release-4.10__arm64-nightly
release-4.10__arm64-stable
release-4.10__arm64-stable
release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10
release-4.11__amd64-nightly
release-4.11__amd64-nightly
release-4.11__amd64-stable
release-4.11__amd64-stable
release-4.11__arm64-nightly-4.11-upgrade-from-stable-4.10
release-4.11__arm64-nightly
release-4.11__arm64-nightly
release-4.11__arm64-stable
release-4.11__arm64-stable
release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11
release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.12
release-4.12__amd64-nightly
release-4.12__amd64-nightly
release-4.12__amd64-stable-4.12-upgrade-from-stable-4.11
release-4.12__amd64-stable
release-4.12__amd64-stable
release-4.12__arm64-nightly-4.12-upgrade-from-stable-4.11
release-4.12__arm64-nightly-4.12-upgrade-from-stable-4.12
release-4.12__arm64-nightly
release-4.12__arm64-nightly
release-4.12__arm64-stable
release-4.12__arm64-stable
release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12
release-4.13__amd64-nightly
release-4.13__amd64-nightly
release-4.13__arm64-nightly
release-4.13__arm64-nightly
```

### CODE REVIEW

**Comment by yunjiang29** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-aws-ipi-byo-kms-etcd-encryption-p2-f14

**Comment by yunjiang29** [comment]
@jianlinliu PTAL

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -96,6 +96,29 @@ tests:
...
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-byo-kms-etcd-encryption
+- as: aws-ipi-byo-kms-etcd-encryption-p3-f28-destructive
+  cron: 18 15 7 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips and @destructive'
+      TAG_VERSION: '@4.11'
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:118]: 4.12?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 2]
```diff
@@ -96,6 +96,29 @@ tests:
...
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-alibabacloud-ipi-private
+- as: aws-ipi-byo-kms-etcd-encryption-p2-f14
+  cron: 32 13 8,24 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips'
+      TAG_VERSION: '@4.11'
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:106]: 4.12?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__arm64-nightly-4.11-upgrade-from-stable-4.10.yaml [hunk 3]
```diff
@@ -109,6 +109,26 @@ tests:
...
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:arm64-target
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+      UPGRADE_CHECK_RUN_TAGS: '@arm64 and @upgrade-check and @4.10 and @aws-ipi and
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__arm64-nightly-4.11-upgrade-from-stable-4.10.yaml:124]: 4.11?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml [hunk 4]
```diff
@@ -56,6 +56,29 @@ resources:
...
 tests:
+- as: aws-ipi-byo-kms-etcd-encryption-p2-f14
+  cron: 17 20 14,30 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips'
+      TAG_VERSION: '@4.12'
+      TEST_ADDITIONAL: Network_Observability
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml:67]: I guess no need this line

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 5]
```diff
@@ -168,6 +168,29 @@ tests:
...
     workflow: cucushift-installer-rehearse-aws-ipi-byo-iam-role
+- as: aws-ipi-byo-kms-etcd-encryption-p2-f14
+  cron: 23 18 11,26 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @fips'
+      TAG_VERSION: '@4.13'
+      TEST_ADDITIONAL: Network_Observability
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:179]: I guess no need this line

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml [hunk 6]
```diff
@@ -143,6 +143,29 @@ tests:
...
     workflow: cucushift-installer-rehearse-aws-ipi-byo-iam-role
+- as: aws-ipi-byo-kms-etcd-encryption-p2-f14
+  cron: 32 22 15,31 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-openshiftsdn and not @fips'
+      TAG_VERSION: '@4.11'
+      TEST_ADDITIONAL: Network_Observability
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml:154]: I guess no need this line

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-stable.yaml [hunk 7]
```diff
@@ -44,6 +44,29 @@ resources:
...
 tests:
+- as: aws-ipi-byo-kms-etcd-encryption-p2-f14
+  cron: 44 9 16,31 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-openshiftsdn and not @fips'
+      TAG_VERSION: '@4.11'
+      TEST_ADDITIONAL: Network_Observability
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-stable.yaml:55]: I guess no need this line

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 8]
```diff
@@ -96,6 +96,29 @@ tests:
...
+      TEST_ADDITIONAL: Network_Observability
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:107]: I guess no need this line

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 9]
```diff
@@ -925,7 +947,7 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       E2E_RUN_TAGS: '@aws-ipi and @proxy and @network-ovnkubernetes and not @fips
         and @destructive'
-      TAG_VERSION: '@4.11'
+      TAG_VERSION: '@4.13'
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:950]: @jianlinliu another incorrect tag version, fixed.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 10]
```diff
@@ -92,10 +92,32 @@ tests:
     cluster_profile: alibabacloud-qe
     env:
       E2E_RUN_TAGS: '@alibaba-ipi and @network-ovnkubernetes and @destructive'
-      TAG_VERSION: '@4.13'
+      TAG_VERSION: '@4.12'
```
Comment (yunjiang29) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:95]: @jianlinliu another incorrect tag version, fixed.

**Comment by yunjiang29** [comment]
@jianlinliu thanks for you checking, all your comments are addressed, PTAL, thanks.

**Comment by yunjiang29** [comment]
@liangxia PTAL

**Pull request merged**